easybuild/easyblocks/o/openblas.py

```diff
@@ -26,10 +26,18 @@ def configure_step(self):
             'USE_OPENMP': '1',
             'USE_THREAD': '1',
         }
-        if LooseVersion(self.version) < LooseVersion('0.3.6') and get_cpu_architecture() == POWER:
+
+        TARGET = 'TARGET'
```

Comment:
@lexming Please move declaring the constant outside of the class (so at line 15, with 2 blank lines after it)

Reply:
that's indeed much prettier, fixed :+1: